# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogGps.h
# SbgLogGpsPos
uint32 timeStamp
uint32 status
uint32 timeOfWeek
float64 latitude
float64 longitude
float64 altitude
float32 undulation
float32 latitudeAccuracy
float32 longitudeAccuracy
float32 altitudeAccuracy
uint8 numSvUsed
uint16 baseStationId
uint16 differentialAge